src/components/Dashboard/FilingHistoryList/HeaderActions.vue

```diff
@@ -0,0 +1,182 @@
+<template>
```

Comment:
This is the common header actions used by FilingTemplate.

This is the fallback "actions" if a filing doesn't override the slot.